ui.apps/src/main/content/jcr_root/apps/venia/clientlibs/common/js/CommerceGraphqlApi.js

```diff
@@ -66,18 +66,21 @@
                 throw new Error(JSON.stringify(response.errors));
             }
             let items = response.data.products.items;
-            return items.map(item => {
+
+            let productsMedia = {};
+
+            items.forEach(item => {
                 let variants = item.variants;
                 if (variants.length > 0) {
 
-                    let variant = variants.find(v => v.product.sku === productData[item.name]);
-                    return variant ? {[variant.product.sku]: imageUrlPrefix + variant.product.media_gallery_entries[0].file} : {};
-
-                } else {
-                    return {};
+                    let skus = productData[item.name];
```

Comment:
This was a bit confusing to me.

Reply:
I forgot to update the comment for this function. 

`productData` it's a "map" in the shape of `{productName:[list-of-skus]}`. The reason there is a list of skus is that if you have two variants of the same product in the cart you have the same name for multiple skus.